src/var.cpp

```diff
@@ -796,6 +796,13 @@ static void jitc_raise_placeholder_error(const char *func, uint32_t index) {
     );
 }
 
+static bool is_default_mask(Variable *v) {
+    return (VarType) v->type == VarType::Bool &&
+        (JitBackend) v->backend == JitBackend::LLVM &&
+        !v->data && !v->literal &&
+        strstr(v->stmt, "%end") != nullptr;
```

Comment:
We actually know the full statement for the default mask variable, so why are we only doing a substring search instead of a full comparison?